package.json

```diff
@@ -4,21 +4,28 @@
   "private": true,
   "dependencies": {
     "@date-io/date-fns": "^1.3.13",
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
     "@loadable/component": "^5.15.0",
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.60",
     "@material-ui/pickers": "^3.3.10",
+    "@mui/lab": "^5.0.0-alpha.48",
+    "@mui/material": "^5.0.1",
+    "axios": "^0.21.4",
     "cra-template": "1.1.2",
     "date-fns": "^2.24.0",
     "history": "^5.0.1",
     "json-server": "^0.16.3",
     "lodash": "^4.17.21",
+    "material-ui-search-bar": "^1.0.0",
```

Comment:
Why is this separate package isn't material UI enough to handle this ??

Reply:
Used by whoever worked on order history on user side. I think so too that it's unnecessary. I've done the filtration in separate branch whose PR I will raise after this one. In which I reused my form component and did nothing extra. I'm removing it.